entities/moduletools/init_params.go

```diff
@@ -23,19 +23,22 @@ type ModuleInitParams interface {
 	GetAppState() interface{}
 	GetLogger() logrus.FieldLogger
 	GetConfig() config.Config
+	// GetConfigPtr() used to be a pointer to a config.Config to be used for
+	// runtime overrides initialization.
+	GetConfigPtr() *config.Config
```

Comment:
can we replace it with `GetConfig()` return `*config.Config` instead? I know we have fix it in so many places. But having one api to get a config is better than two apis for same purpose? More confusion for long-term I think. wdyt?